contracts/contracts/RequestManager.sol

```diff
@@ -252,15 +252,13 @@ contract RequestManager is Ownable {
 
         address nextActor;
         uint256 minValue;
-        uint256 periodExtension;
+        uint256 periodExtension = challengePeriodExtension;
```

Comment:
Can you add an assertion somewhere that catches this?

Reply:
What exctly do you have in mind to be asserted?

Reply:
Basically make sure we test this, so an assertion for the extension of the challenge period in that case.

Reply:
Yes, I agree with @palango here, it'd be nice to say "new termination time is _at least this much_ later than old termination time".

Reply:
I can also do it in the contract instead of a test.

Reply:
There is `test_claim_period_extension()` which simply didn't cover that path. I extended the test.

Reply:
Perfect, thanks!